src/fsharp/tast.fs

```diff
@@ -62,6 +62,8 @@ type ValInline =
     | PseudoVal
     /// Indicates the value is inlined but the .NET IL code for the function still exists, e.g. to satisfy interfaces on objects, but that it is also always inlined 
     | Always
+    /// Indicates the value will be inlined be inlined by the .NET runtime
+    | Aggressive
```

Comment:
I'm not convinced we need to change the TAST (or Pickle) at all?

I think all you need to do is use the TAST value ``ValInline.Never`` if the attribute is present.